packages/utils/src/multiaddrToIpNet.ts

```diff
@@ -0,0 +1,32 @@
+import { multiaddr } from '@multiformats/multiaddr'
+import { convertToIpNet } from '@multiformats/multiaddr/convert'
+import type { IpNet } from '@chainsafe/netmask'
+import type { Multiaddr } from '@multiformats/multiaddr'
+
+/**
+ * Converts a multiaddr string or object to an IpNet object.
+ * If the multiaddr doesn't include /ipcidr/32, it will encapsulate with /ipcidr/32.
+ *
+ * @param {string | Multiaddr} ma - The multiaddr string or object to convert.
+ * @returns {IpNet} The converted IpNet object.
+ * @throws {Error} Throws an error if the multiaddr is not valid.
+ */
+export function multiaddrToIpNet (ma: string | Multiaddr): IpNet {
```

Comment:
This function is only used in the `libp2p` module so please define it there in the [src/connection-manager/utils.ts](https://github.com/libp2p/js-libp2p/blob/main/packages/libp2p/src/connection-manager/utils.ts) file.  Please also move the tests to that module.

If we later find that we need it elsewhere we can move it to the utils package, but right now there's no need to expand the public API.

Reply:
I've moved the multiaddrToIpNet function to the connection-manager/utils.ts file as requested and relocated the corresponding tests. I initially placed it in the general utils package for broader use, but I understand your point about keeping the API focused for now. It's done!